src/core/init.ts

```diff
@@ -68,4 +72,6 @@ export function init() {
     higlassRegister({ dataFetcher: CSVDataFetcher, config: CSVDataFetcher.config }, { pluginType: 'dataFetcher' });
     higlassRegister({ dataFetcher: RawDataFetcher, config: RawDataFetcher.config }, { pluginType: 'dataFetcher' });
     higlassRegister({ dataFetcher: BBIDataFetcher, config: BBIDataFetcher.config }, { pluginType: 'dataFetcher' });
+
+    once = true;
```

Comment:
I thought it made sense to call this function once, unless there is some reason this is not desired ...

Reply:
I agree!